.github/workflows/ci.yml

```diff
@@ -11,7 +11,9 @@ env:
   RUST_MIN_VER: "1.70"
   # List of packages that will be checked with the minimum supported Rust version.
   # This should be limited to packages that are intended for publishing.
-  RUST_MIN_VER_PKGS: "-p parley -p fontique"
+  PRIMARY_PKGS: "-p parley -p fontique"
```

Comment:
You don't use `PRIMARY_PKGS` below, but you do use `PUBIC_PKGS` ...

Is the purpose of renaming this away from `RUST_MIN_VER_PKGS` to denote which are published and not?

Reply:
Ee're running `check/build/test/doc` etc over `parley` and `fontique` crates with `--features std` and `--features libm`, neither of which work with the `tiny_skia_render` example crate because it doesn't have those features. So we can't use `--workspace` to run those checks. So I'm using this "primary packages" variable to do specify "parley and fontique".